openkfw/TruBudget
# 1800 Fix header padding issue when dropdown/dialog opens

### Checklist




### Description
- fix issue when header is moving to the left when dropdown or dialog window is opened

### How to test

1. login as any user
2. try to change pagination limit by opening dropdown in project card view
3. right navigation should not move

Closes #1800
